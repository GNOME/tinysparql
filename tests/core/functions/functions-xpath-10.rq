PREFIX ex: <http://example/>
PREFIX fn: <http://www.w3.org/2005/xpath-functions#>

# Tests the replace function.
# The punctuation shall be removed to alphabetize
# the names of the persons corectly.
SELECT ?name
WHERE
{
    ?s  ex:name   ?name .
}
ORDER BY fn:replace(?name, "^[ ,'\".]+", "")
