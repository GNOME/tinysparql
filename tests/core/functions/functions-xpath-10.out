"Alice Hyatt"
"Amelie Colbert"
" Dell Quesada "
"Emily Schwartz"
"Fritz Fischer"
"'Jule Wolk'"
" Kathlene Beckmann "
" Maximilian Muster "
"  Mia Simpson  "
" Torie Baumeister "
